src/Microsoft.ML.Transforms/Text/NgramTransform.cs

```diff
@@ -334,20 +334,20 @@ private static void AssertValid(IHostEnvironment env, int[] counts, ImmutableArr
             env.Assert(count == pool.Count);
         }
 
-        private static NgramIdFinder GetNgramIdFinderAdd(IHostEnvironment env, int[] counts, ImmutableArray<int> lims, SequencePool pool, bool requireIdf)
+        private static NgramIdFinder GetNgramIdFinderAdd(IHostEnvironment env, int[] counts, IReadOnlyList<int> lims, SequencePool pool, bool requireIdf)
```

Comment:
>IReadOnlyList [](start = 93, length = 13)

curious, why change this?@

Reply:
For consistency with tree's public interface and other places.

---
In reply to: [264022662](https://github.com/dotnet/machinelearning/pull/2898#discussion_r264022662) [](ancestors = 264022662)